### DISCUSSION TITLE
Why catppuccin mocha?

### DISCUSSION THREAD

**Discussion by kasergus** [Meowix-Linux/Meowix-ISO#10] [General]
Why meowix linux developers choosed catppuccin mocha theme? Why not macchiato or frappe?

**Comment by Nekosis** [comment]
Good question! From what we've seen, Mocha seems to be the most widely supported variant of Catppuccin. While the goal for Catppuccin's developers is to make sure all themes are supported equally, Mocha seems to be the most popular of the flavors, and thus, more themes are specifically built using it rather than the other flavors.
